src/server/http/max_payload_size.test.js

```diff
@@ -65,6 +65,6 @@ test('fails with 400 if payload size is larger than default and route config all
     payload: { data: Array(250).fill('+').join('') },
   });
 
-  expect(resp.statusCode).toBe(400);
+  expect(resp.statusCode).toBe(413);
```

Comment:
This seems like a potentially unrelated change that should be in a separate PR if it's really necessary.

Reply:
Hapi changed how max body size errors were handled to return the correct error code. Talked to Tyler about this and we thought this would be a pretty safe change. I checked that client-side code does not break from this change.